tracer/test/test-applications/azure-functions/Directory.Build.props

```diff
@@ -0,0 +1,44 @@
+<Project>
```

Comment:
Are you unable to use the Directory.Build.props file that's higher up? This seems like a straight copy/paste

Reply:
I suppose I could add a condition to the higher up file to not force this to be an exe, but that was why I made this.

Reply:
I tried and failed to add conditionals to the higher level build props. The host can't pickup the functions when OutputType is specified.